### PR TITLE
DOC: fix three minor doc issues

### PR DESCRIPTION
**Pull request by iboyd-ansys**
Fixes for issues:

* #131 
* #132 
* #133 

All of these are effectively one liners, though 132 turned out to be a different fix from the one requested. Also, #132 has a larger impact because it requires the Sphinx gallery examples to be regenerated.

### CODE REVIEW

File: doc/source/examples/00-systemcoupling/images/sphx_glr_parametric_sweep_vel_001.png
Comment (iboyd-ansys) [inline doc/source/examples/00-systemcoupling/images/sphx_glr_parametric_sweep_vel_001.png:1]: @ochernuk I don't know why results have changed for velocities <= 20 m/s. This was run with 23.1 as before.

File: doc/source/index.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ System Coupling is software for solving multiphysics problems by connecting inde
 solvers and coordinating the exchange of the solution data. This enables accurate capturing of the
 complex interactions between physical models, typically simulated in separate solvers.
 
-System Coupling connects to several existing Ansys physics solvers, including Ansys CFX,
+System Coupling connects to several existing Ansys physics solvers, including Ansys Fluent, Ansys CFX,
```
Comment (iboyd-ansys) [inline doc/source/index.rst:23]: Issue #131

File: doc/source/index.rst [hunk 2]
```diff
@@ -61,7 +61,7 @@ local copy of System Coupling.
 .. TODO Add the appropriate link when available.
 
 For more information about System Coupling,
-visit the Ansys System Coupling page on the Ansys website.
+visit the `Ansys System Coupling page <https://www.ansys.com/products/system-coupling>`_ on the Ansys website.
```
Comment (iboyd-ansys) [inline doc/source/index.rst:64]: Issue #133

File: examples/00-systemcoupling/parametric_sweep_vel.py [hunk 3]
```diff
@@ -213,7 +213,7 @@ def plot(working_dir, x, y):
 # are used to fill in the corresponding values of the ``y`` array.
 # Finally, call the ``plot()`` function to generate a plot from the arrays.
 
-x = np.array([5.0, 10.0, 15.0, 20.0, 25.0])
+x = np.array([5.0, 10.0, 15.0, 20.0])
```
Comment (iboyd-ansys) [inline examples/00-systemcoupling/parametric_sweep_vel.py:216]: Issue #132

File: doc/source/examples/00-systemcoupling/images/sphx_glr_parametric_sweep_vel_001.png
Comment (iboyd-ansys) [inline doc/source/examples/00-systemcoupling/images/sphx_glr_parametric_sweep_vel_001.png:1]: There was a general issue with all points - not just the last. Had missed the y-axis scaling on the plot.

Issue is now resolved.

Comment (iboyd-ansys) [reply doc/source/examples/00-systemcoupling/images/sphx_glr_parametric_sweep_vel_001.png:1]: **NB** All the changes under `doc/source/examples/00-systemcoupling` are generated so don't need to be reviewed in detail. This is the only one where there is something interesting to see.

**Review by ochernuk — APPROVED** [review]

**Pull request merged**